examples_tests/tests.py

```diff
@@ -214,14 +214,17 @@ def setUpClass(cls):
         cls.temp_dir = tempfile.mkdtemp()
         if not config.get('skip-install', False):
             ip = config.get('ip', None)
-            if ip is None:
+            if not ip:
                 cls.testbed_ip = 'localhost'
                 cls.testbed_port = '8022'
                 cls.vm_process = _start_snappy_testbed(
                     cls.temp_dir, cls.testbed_port)
             else:
                 cls.testbed_ip = ip
-                cls.testbed_port = config.get('port', '22')
+                port = config.get('port', None)
```

Comment:
Because when the argument is not passed docopt sets the port to None, and {'port': None}.get('port', 22) returns None, not 22.